src/app/components/card/card.component.ts

```diff
@@ -13,6 +13,9 @@ export class CardValue {
   avatar?: string;
   link?: string;
   menuItem?: any;
+  selfLink?: string;
```

Comment:
These three are very work item specific. Take one object called **extraData** or something like that and have these three inside it.